### PR TITLE
refactor: rename standalone to kestrel

### PR DESCRIPTION
**Pull request by weichweich**
## fixes https://github.com/KILTprotocol/ticket/issues/2794

* i decided to remove the mashnet naming entirely. Mashnet is the old thing from the very beginning.
* the mashnet-node was renamed to standalone-node since it's not specific to kestrel (there could be a second runtime executed in with this node, like we do it for the parachain-node)
* change the impl-name to `kilt-peregrine`
## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

**Review by ntn-x2 — COMMENTED** [review]
I see quite few occurrences of mashnet with a quick search function.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# KILT-node &middot; [![tests](https://gitlab.com/kiltprotocol/mashnet-node/badges/develop/pipeline.svg)](https://gitlab.com/kiltprotocol/mashnet-node/-/commits/develop)
+# KILT-node &middot; [![tests](https://gitlab.com/kiltprotocol/kilt-node/badges/develop/pipeline.svg)](https://gitlab.com/kiltprotocol/kilt-node/-/commits/develop)
```
Comment (ntn-x2) [inline README.md:1]: 🥶🥶🥶

**Review by ntn-x2 — COMMENTED** [review]
`.gitlab-ci.yml` still has a `DOCKER_HUB_STANDALONE: "kiltprotocol/mashnet-node"` which should be renamed.

**Comment by weichweich** [comment]
> `.gitlab-ci.yml` still has a `DOCKER_HUB_STANDALONE: "kiltprotocol/mashnet-node"` which should be renamed.

This is the image on dockerhub. I would keep this name. Not sure what depends on this naming.

File: runtimes/peregrine/src/lib.rs [hunk 2]
```diff
@@ -90,7 +90,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("mashnet-node"),
-	impl_name: create_runtime_str!("mashnet-node"),
+	impl_name: create_runtime_str!("kilt-peregrine"),
```
Comment (ntn-x2) [inline runtimes/peregrine/src/lib.rs:93]: Not sure I like this... EIther we don't change it at all, or we change both. What's the reason for changing only the implementation?

Comment (weichweich) [reply runtimes/peregrine/src/lib.rs:93]: The idea was that there is at least one difference in the naming. spec name might be the same but at least the impl name is different.

Comment (ntn-x2) [reply runtimes/peregrine/src/lib.rs:93]: I think spec name should also be different. That's the assumption the type definitions do as well. I think impl name is probably a different implementation of the Peregrine runtime. But the spec name indicates the runtime. If Peregrine is not mashnet, then the spec name should not be mashnet either.

Comment (weichweich) [reply runtimes/peregrine/src/lib.rs:93]: Yes the only reason I'm not changing the spec name is, that I'm not 100% sure about the consequences

Comment (weichweich) [reply runtimes/peregrine/src/lib.rs]: Ok let's do this as part of another PR once we researched all the consequences of renaming the spec.

[8adcaa4](https://github.com/KILTprotocol/kilt-node/pull/554/commits/8adcaa454c4ba8fe4dc2de08634695d67d4c4861)

**Review by ntn-x2 — APPROVED** [review]
🐦‍⬛🐦‍⬛🐦‍⬛

**Comment by weichweich** [comment]
As a cherry on top, here is the `subwasm diff` report:

<details>
  <summary>Develop branch Peregrine Diff</summary>

```
!!! THE SUBWASM REDUCED DIFFER IS EXPERIMENTAL, DOUBLE CHECK THE RESULTS !!!
No change detected
SUMMARY:
- Compatible.......................: true
- Require transaction_version bump.: false

!!! THE SUBWASM REDUCED DIFFER IS EXPERIMENTAL, DOUBLE CHECK THE RESULTS !!!
```

</details>

<details>
  <summary>Develop branch Spiritnet Diff</summary>

```
!!! THE SUBWASM REDUCED DIFFER IS EXPERIMENTAL, DOUBLE CHECK THE RESULTS !!!
No change detected
SUMMARY:
- Compatible.......................: true
- Require transaction_version bump.: false

!!! THE SUBWASM REDUCED DIFFER IS EXPERIMENTAL, DOUBLE CHECK THE RESULTS !!!
```

</details>

**Pull request merged**